libs/application/templates/car-recycling/src/lib/answerValidationSections/vehicles.ts

```diff
@@ -0,0 +1,19 @@
+import { Application } from '@island.is/application/types'
+
+import { AnswerValidationConstants } from '../../shared/constants'
+import { buildError } from './utils'
+import { VehicleMiniDto } from '@island.is/clients/vehicles'
+import { errorMessages } from './../messages'
+
+export const vehicles = (newAnswer: unknown, application: Application) => {
```

Comment:
I would encourage you to use zod validation for this case. If it doesnt seem feasible I would like to give it a try as we believe currently that most cases should be possible to tackle with zod. It would take me a day or two to free up my backlog tho. We are trying to phase out answer validators, but it might not be possible.

Reply:
We have found the solution for this and removed the answer validators, so it is part of this PR